SQL/PostgreSQL.sublime-syntax

```diff
@@ -66,6 +66,14 @@ contexts:
         2: punctuation.section.brackets.end.psql
       set: maybe-group
 
+###[ COMMENTS ]################################################################
+
+  double-dash-comments:
+    - meta_append: true
```

Comment:
As the context contains just this one rule in both SQL (basic) and MySQL, we probably don't need to append, just replace again to apply SQL (basic) pattern.

```suggestion
```